.github/workflows/build-id.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Harden Runner
-        uses: step-security/harden-runner@cba0d00b1fc9a034e1e642ea0f1103c282990604 # v2.5.0
+        uses: step-security/harden-runner@8ca2b8b2ece13480cda6dacd3511b49857a23c09 # v2.5.1
```

Comment:
why are these getting updated?

Reply:
I think it was showing those as updated since I didn't rebase on top of the changes after fixing the commits for DCO issue